docs/telemetry.md

```diff
@@ -0,0 +1,206 @@
+# Telemetry labs
+
+## gNMIc Prometheus Grafana Lab
+
+### Lab Topology
+
+![lab topology](assets/img/aclabs-telemetrylab-A.png)
+
+### Demo Details
+
+!!! Success "Container Requirements"
+
+    CPUs: 8
+    Memory: 32 GB
+    Storage: 64 GB
+
+### CPU Architechture
+
+- Only x86 CPU architecture is supported!
+- ARM support coming soon!
+
+### How To Run The Workshop
+
+???+ Tip "Wait until cEOS-lab image will be imported"
+
+    You can confirm if image was imported correctly with `docker image ls`.
+    In some rare cases the cEOS-lab image may fail to be downloaded.
+    For example due to incorrecte token or other API failure on arista.com.
+    In that case you can upload the image manually by right-clicking the Explorer tab of the VSCode interface.
+
+To run the labs in this workshop, you can use one of the following options:
+
+- Build you own Ubuntu VM from scratch. (Recommended!)
+  - This option will allow you to experience the entire cLab enviroment build process from the very start, without any pre-installed dependancies
+  - Requirements
+    - Ubuntu LTS 22.04 or later
+    - 32 GB RAM and 8 vCPUs
+- Start the [Github Codespace](https://codespaces.new/noredistribution/aclabs/tree/main?quickstart=1&devcontainer_path=.devcontainer%2Fgnmic-prometheus-grafana%2Fdevcontainer.json) from this repository. (Fastest!)
+- Open the lab dev container locally on your laptop with Docker Desktop or sever with Docker CE:
+  - download one of the prepackage AVD+Python+Containerlab containers from [aclabs packages](https://github.com/aristanetworks/aclabs/pkgs/container/aclabs%2Flab-base)
+  - when running the lab locally, you must set all required environment variables on your machine
+
+### Run it in Codespaces
```

Comment:
Combine this with the previous section, refer to the Quickstart and only leave what is relevant for this specific lab, like the screenshot with the 3001 port.